### ISSUE TITLE
Postman application turns black in full screen mode in Docker container

### ISSUE THREAD

**Issue by OlgaUtochka** [a85/POSTMan-Chrome-Extension#7680]
**Describe the bug**
After the last update I cannot resize the window or use Postman in full screen mode because the window turns black after any action. I work in a Docker container, which may be the reason for this issue, though I haven't encountered this problem in previous versions. 

**To Reproduce**
1. Open the Postman application. 
2. Resize the window (or open it in a full screen)

**Screenshots**

![Screenshot at 2019-11-20 11-31-26](https://user-images.githubusercontent.com/44089660/69236580-b6981000-0b8b-11ea-9c83-7a19c5c3f5c7.png)

**App information:**
 - Postman Version: 7.11.0
 - OS: Ubuntu 18.04.2 LTS (linux-4.15.0-66-generic on x64)

**Comment by harryi3t** [comment]
@OlgaUtochka Could you share with us the logs `View -> Developer -> View logs in File Explorer`?
Also, will it be possible to share the docker setup with us so that we can take a look?

**Comment by OlgaUtochka** [comment]
Sorry, I cannot share the docker setup at the moment. We use NoMachine server for Docker. I've checked Postman in another container and it worked fine (this container was created to test Postman, no other apps except for Mozilla Firefox were installed). 

By the way, two days ago I was able to work in Postman if a window was small enough, but now it turns black immediately, even in a small window :( The only thing I can see before it gets black is the small button on the top (like it's connecting/updating the info). And sometimes the app just crashes. I thought that it could be connected to other running apps or opened browser, but I cannot use Postman even if all other apps are closed. 

Here are the logs:
[main.log](https://github.com/postmanlabs/postman-app-support/files/3879152/main.log)
[pluginHost.log](https://github.com/postmanlabs/postman-app-support/files/3879153/pluginHost.log)
[renderer-auth.log](https://github.com/postmanlabs/postman-app-support/files/3879154/renderer-auth.log)
[renderer-console.log](https://github.com/postmanlabs/postman-app-support/files/3879155/renderer-console.log)
[renderer-requester.log](https://github.com/postmanlabs/postman-app-support/files/3879156/renderer-requester.log)
[renderer-runner.log](https://github.com/postmanlabs/postman-app-support/files/3879157/renderer-runner.log)
[renderer-shared.log](https://github.com/postmanlabs/postman-app-support/files/3879158/renderer-shared.log)
[renderer-shell.log](https://github.com/postmanlabs/postman-app-support/files/3879159/renderer-shell.log)

Hope there may be some solution to this problem.

**Comment by twitharshil** [comment]
Hey @OlgaUtochka 

The platform layer for the postman application doesn't work very well with GPU drivers available on your system. 

On the recent versions of the postman application, we added an option to try disabling hardware acceleration on your machine to make it work. Read more about it [here](https://learning.postman.com/docs/getting-started/settings/#hardware-acceleration). 

If you are still on the Postman v7 application, try launching postman along with passing the `--disable-gpu` command-line argument and see if that helps!

Closing this as a duplicate https://github.com/postmanlabs/postman-app-support/issues/5955
